wom/src/main/scala/wom/types/WomIntegerLike.scala

```diff
@@ -5,4 +5,7 @@ object WomIntegerLike {
   implicit class EnhancedLong(val long: Long) extends AnyVal {
     def inIntRange: Boolean = long >= Int.MinValue && long <= Int.MaxValue
   }
+  implicit class EnhancedDouble(val double: Double) extends AnyVal {
+    def inIntRange: Boolean = double >= Int.MinValue && double <= Int.MaxValue
```

Comment:
As well as not seeing this used anywhere else, I'm a little skeptical of this as something which is comparing `Double`s and `Int`s?